app.rb

```diff
@@ -0,0 +1,192 @@
+require './student'
+require './teacher'
+require './book'
+require './rental'
+
+class App
+  def initialize
+    @books = []
+    @people = []
+    @rentals = []
+  end
+
+  def start_menu
+    puts 'Please choose an option by entering a number:'
+    puts '1. List all books'
+    puts '2. List all people'
+    puts '3. Create a person'
+    puts '4. Create a book'
+    puts '5. Create a rental'
+    puts '6. List all rentals for a given person ID'
+    puts '7. Exit'
+  end
+
+  def start
+    start_menu
+    @option = gets.chomp
+
+    case @option.to_i
+    when 1 # List all books
+      list_all_books
+    when 2 # List all people
+      list_all_people
+    when 3 # Create a person
+      create_person
+    when 4 # Create a book
+      create_book
+    when 5 # Create a rental
+      add_rental
+    when 6 # List all rentals a given person ID
+      list_rentals_by_person
+    else # Exit
+      puts 'Goodbye :-)'
+    end
+  end
```

Comment:
- [ ] Kindly move everything about starting the app, the code that asks the user for the initial choice of options (or to quit) to the `main.rb` file. This is to achieve the `Single-responsibility [Principle](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/lessons/solid.md#best-practices-and-solid)`